### PR TITLE
Add personal handwired Planck clone

### PR DESCRIPTION
**Pull request by BifbofII**
## Description

Config and keymap for a handwired Planck clone.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by BifbofII** [comment]
All references to Planck except for calling it a 'planck-like' build in the readme are now removed and renamed.

File: keyboards/handwired/ortho_brass/config.h [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID 0xFEED
+#define PRODUCT_ID 0x6060
+#define DEVICE_VER 0x0001
+#define MANUFACTURER BifbofII
+#define PRODUCT "Ortho Brass"
+#define DESCRIPTION "Handwired Ortho 4x12 Keyboard"
```
Comment (drashna) [inline keyboards/handwired/ortho_brass/config.h:27]:
```suggestion
#define PRODUCT Ortho Brass
```

File: keyboards/handwired/ortho_brass/config.h [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+#define MATRIX_COL_PINS \
+    { F4, F7, F5, F1, C7, F0, B1, B0, F6, B6, B2, B3 }
+#define UNUSED_PINS \
+    { B0, B1, B2, F0, F1, F4, D4, D5, E6 }
+
+/* COL2ROW or ROW2COL */
+#define DIODE_DIRECTION COL2ROW
+
+/* Debounce reduces chatter (unintended double-presses) - set 0 if debouncing is not needed */
+#define DEBOUNCING_DELAY 5
```
Comment (drashna) [inline keyboards/handwired/ortho_brass/config.h:45]:
```suggestion
#define DEBOUNCE 5
```

File: keyboards/handwired/ortho_brass/ortho_brass.c [hunk 3]
```diff
@@ -0,0 +1,10 @@
+#include "ortho_brass.h"
```
Comment (drashna) [inline keyboards/handwired/ortho_brass/ortho_brass.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/handwired/ortho_brass/ortho_brass.h [hunk 4]
```diff
@@ -0,0 +1,31 @@
+#pragma once
```
Comment (drashna) [inline keyboards/handwired/ortho_brass/ortho_brass.h:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/handwired/ortho_brass/rules.mk [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+UNICODE_ENABLE = no         # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+RGBLIGHT_ENABLE = no        # Enable WS2812 RGB underlight.
+API_SYSEX_ENABLE = no
+
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+
+LAYOUTS = ortho_4x12
+LAYOUTS_HAS_RGB = no
```
Comment (drashna) [inline keyboards/handwired/ortho_brass/rules.mk:27]:
```suggestion
# Build Options
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
MOUSEKEY_ENABLE = no        # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
NKRO_ENABLE = yes           # N-key Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable RGB underlight.

# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend

LAYOUTS = ortho_4x12

RGBLIGHT_SUPPORTED = no
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/handwired/ortho_brass/keymaps/default/keymap.c [hunk 6]
```diff
@@ -0,0 +1,239 @@
...
+            if (record->event.pressed) {
+                layer_off(_PLOVER);
+            }
+            return false;
+            break;
+    }
+    return true;
+}
+
+void dip_switch_update_user(uint8_t index, bool active) {
```
Comment (drashna) [inline keyboards/handwired/ortho_brass/keymaps/default/keymap.c:218]:
```suggestion
bool dip_switch_update_user(uint8_t index, bool active) {
```

File: keyboards/handwired/ortho_brass/keymaps/default/keymap.c [hunk 7]
```diff
@@ -0,0 +1,239 @@
...
+        case 0: {
+            if (active) {
+                layer_on(_ADJUST);
+            } else {
+                layer_off(_ADJUST);
+            }
+            break;
+        }
+    }
+}
```
Comment (drashna) [inline keyboards/handwired/ortho_brass/keymaps/default/keymap.c:229]:
```suggestion
    return true;
}
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

**Review by tzarc — APPROVED** [review]

**Pull request merged**